### PR TITLE
scripts: Make user could use their own rke2 repo

### PR DESCRIPTION
**Pull request by Vicente-Cheng**

### CODE REVIEW

File: Dockerfile.dapper [hunk 1]
```diff
@@ -42,6 +42,9 @@ ENV DAPPER_OUTPUT ./bin ./dist
 ENV DAPPER_DOCKER_SOCKET true
 ENV DAPPER_RUN_ARGS "-v /run/containerd/containerd.sock:/run/containerd/containerd.sock -v harvester-installer-go:/root/go -v harvester-installer-cache:/root/.cache"
 
+# You cloud defined your own rke2 url for iso building fast
+#ENV LOCAL_RKE2_IMAGE_REPO=<your http url for rke2>
```
Comment (bk201) [inline Dockerfile.dapper:46]: Should this be listed in `DAPPER_ENV`?

File: scripts/build-bundle [hunk 2]
```diff
@@ -5,6 +5,14 @@ TOP_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )/.." &> /dev/null && pwd )"
...
 PACKAGE_HARVESTER_OS_DIR="${TOP_DIR}/package/harvester-os"
 PACKAGE_HARVESTER_REPO_DIR="${TOP_DIR}/package/harvester-repo"
+DEFAULT_RKE2_IMAGE_REPO="https://github.com/rancher/rke2/releases/download/"
+if [[ ${LOCAL_RKE2_IMAGE_REPO} ]]
+then
+       RKE2_IMAGE_REPO=${LOCAL_RKE2_IMAGE_REPO}
+else
+       RKE2_IMAGE_REPO=$DEFAULT_RKE2_IMAGE_REPO
+fi
+
```
[the remark this answers is not in the document]
Comment (Vicente-Cheng) [reply scripts/build-bundle]: yap. I also update the latest version.
Maybe looks more simple  : p

File: Dockerfile.dapper (hunk 1 above)
Comment (Vicente-Cheng) [reply Dockerfile.dapper]: yap, done.  thanks for tips!

File: Dockerfile.dapper [hunk 3]
```diff
@@ -42,6 +42,9 @@ ENV DAPPER_OUTPUT ./bin ./dist
 ENV DAPPER_DOCKER_SOCKET true
 ENV DAPPER_RUN_ARGS "-v /run/containerd/containerd.sock:/run/containerd/containerd.sock -v harvester-installer-go:/root/go -v harvester-installer-cache:/root/.cache"
 
+# You cloud defined your own rke2 url to make iso building fast
+ENV DAPPER_ENV RKE2_IMAGE_REPO=
```
Comment (bk201) [inline Dockerfile.dapper:46]: I suggested add the variable just to line 39
```
ENV DAPPER_ENV REPO TAG DRONE_TAG DRONE_BRANCH CROSS RKE2_IMAGE_REPO
```

Comment (Vicente-Cheng) [reply Dockerfile.dapper]: Done. please review again

**Review by tjjh89017 — APPROVED** [review]
LGTM

**Comment by Vicente-Cheng** [comment]
update again for script simplifies, please review

**Review by bk201 — APPROVED** [review]
nice enhancement!

**Pull request merged**